antarest/study/storage/utils.py

```diff
@@ -287,28 +313,27 @@ def get_start_date(
     start_offset = cast(int, config.get("simulation.start"))
     end = cast(int, config.get("simulation.end"))
 
-    starting_month_index = strptime(starting_month, "%B").tm_mon
+    starting_month_index = ENGLISH_MONTHS.index(starting_month.title()) + 1
     target_year = 2000
     while True:
         if leapyear == calendar.isleap(target_year):
             first_day = datetime(target_year, starting_month_index, 1)
-            if first_day.strftime("%A") == starting_day:
+            first_day_name = ENGLISH_DAY_NAMES[first_day.weekday()]
+            if first_day_name == starting_day.title():
```

Comment:
It seems to me it would be clearer to convert `starting_day` to a weekday integer instead:
we can then simply have the comparison:
```python
if first_day.weekday() == starting_day
```